spigot-impl/src/main/java/me/aurium/beetle/spigot/SpigotStringUtil.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Literally shut up i dont care
```

Comment:
I didn't say anything? (yet)

This class is ugly you have to admit.

Reply:
Oh this wasn't directed towards you hahaha

It is ugly, but a sad necessity of spigot development